fr.free.hxc2001.HxCFloppyEmulator.Extension.capslib.yaml

```diff
@@ -2,7 +2,7 @@ id: fr.free.hxc2001.HxCFloppyEmulator.Extension.capslib
 branch: 1-19.08
```

Comment:
you probably also need to have a matching branch for what the main app want.

Reply:
The main app does not specify a branch, and installing it from flathub shows the branch as stable, so am I right in that if I were to remove the branch keyword here, then the branches should match, given that the build process fills the branch in unless otherwise specified?

Reply:
and what is that: https://github.com/flathub/fr.free.hxc2001.HxCFloppyEmulator/blob/master/fr.free.hxc2001.HxCFloppyEmulator.yaml#L8 ?

Reply:
Yes, sorry, should have been clear about that.

Reply:
I maintain my comment here. 

You didn't test this.

Reply:
Worked it out, and yes, tested this time.